next/CMakeLists.txt

```diff
@@ -6,6 +6,7 @@ set(MBGL_ROOT ${PROJECT_SOURCE_DIR}/..)
 
 set_property(GLOBAL PROPERTY USE_FOLDERS ON)
 set_property(GLOBAL PROPERTY PREDEFINED_TARGETS_FOLDER Core)
+option(MAPBOX_MAPS_WITH_OPENGL     "Enable OpenGL renderer." ON)
```

Comment:
can you change this to `MBGL_WITH_OPENGL` and move it to the section of the file that contains the other options?

Reply:
done